org.lflang.tests/src/org/lflang/tests/compiler/LinguaFrancaDependencyAnalysisTest.xtend

```diff
@@ -135,7 +135,7 @@ class LinguaFrancaDependencyAnalysisTest {
         Assertions.assertNotNull(model)
 		
 		var info = new ModelInfo()
-		info.update(model, DefaultErrorReporter.DEFAULT)
+		info.update(model, new DefaultErrorReporter())
```

Comment:
Why create a new instance and not use the static one?

Reply:
Because that object is stateful. Keeping a static instance is basically keeping a global mutable variable which may be accessed concurrently and behave incorrectly, as we don't know who's using it.